README.md

```diff
@@ -1,2 +1,35 @@
 # odtp-manuals
 All manuals, instructions, documentation and information you need to work with ODTP.
+
+## How to serve this documentation in a docker? 
+
+1. `docker build -t optp-docs .`
+2. `docker run -it --rm --p 8000:8000 odtp-docs`
+
+This will serve the documentation in: http://0.0.0.0:8000/odtp-org/
+
+## Development
+
+### How to serve this documentation in local? 
+
+DISCLAIMER: Still to be properly tested
+
+1. `pip install mkdocs-material`
+2. `brew install cairo freetype libffi libjpeg libpng zlib`
+3. `pip install "mkdocs-material[imaging]"`
+4. `mkdocs new`
```

Comment:
fix: I don't think this is necessary, you already did that and got the  `docs` directory.

Reply:
You, are right. Removing `mkdocs new`